test/testcrawl.js

```diff
@@ -47,6 +47,41 @@ describe("Test Crawl", function() {
         });
     });
 
+    it("should obey rules in robots.txt", function(done) {
+
+        this.slow("2s");
+
+        var crawler = new Crawler("127.0.0.1", "/", 3000);
+        crawler.start();
+
+        crawler.on("fetchcomplete", function(queueItem) {
+            queueItem.url.should.not.equal("http://127.0.0.1:3000/forbidden");
+        });
+        crawler.on("complete", function() {
+            done();
+        });
+    });
+
+    it("should be able to disregard rules in robots.txt", function(done) {
+
+        this.slow("2s");
```

Comment:
Takes between 1500 and 1800 ms on my machine. I could probably speed this up by listening to the new `fetchdisallowed` event instead, though